packages/custom-storage/src/sql/SQLStorage.ts

```diff
@@ -318,6 +323,7 @@ export class SQLStorage {
 
     const datasets: string[] = rawDatasets.map((row: { name: string }) => row.name);
 
+    // NOTE: DROP TABLE CASCADE removes dependant views or functions and foreign keys constraints (neither tables nor data)
     return this._sql.query(`
       BEGIN;
         ${datasets.map((datasetName) => `DROP TABLE IF EXISTS ${datasetName};`).join('\n')}
```

Comment:
I didn't know we were removing the datasets themselves! It's ok by now if this is used just for us know, but it looks a bit dangerous to me. I guess we should start to document at least our methods, describing this set of deletions